jira_agile_metrics/config_test.py

```diff
@@ -236,16 +234,14 @@ def test_config_to_options_maximal():
         'https_proxy': 'https://proxy2.local',
         'jira_server_version_check': True
     }
-   
+
     assert options['settings'] == {
         'cycle': [
-            {'name': 'Backlog', 'statuses': ['Backlog'], 'type': 'backlog'},
-            {'name': 'Committed', 'statuses': ['Next'], 'type': 'accepted'},
-            {'name': 'Build', 'statuses': ['Build'], 'type': 'accepted'},
-            {'name': 'Test',
-                'statuses': ['Code review', 'QA'],
-                'type': 'accepted'},
-            {'name': 'Done', 'statuses': ['Done'], 'type': 'complete'}
+            {'name': 'Backlog', 'statuses': ['Backlog']},
+            {'name': 'Committed', 'statuses': ['Next']},
+            {'name': 'Build', 'statuses': ['Build']},
+            {'name': 'Test', 'statuses': ['Code review', 'QA']},
+            {'name': 'Done', 'statuses': ['Done']}
```

Comment:
Think this is OK - I can see you've taken out the `StatusTypes` concept which as per above is probably not all that useful. We should make sure we remove the enum as well if it's unused (and if it is used - why?)

Reply:
Yes, as described in https://github.com/DeloitteDigitalUK/jira-agile-metrics/pull/27#issuecomment-636308642 I think `StatusTypes` in not needed.